code/game/objects/items/weapons/stock_parts.dm

```diff
@@ -6,7 +6,14 @@
 	icon_state = "RPED"
 	item_state = "RPED"
 	w_class = WEIGHT_CLASS_HUGE
-	can_hold = list(/obj/item/stock_parts)
+	can_hold = list(
+		/obj/item/stock_parts,
+		// Added separately in Initialize to avoid picking up unwanted
+		// subtypes.
+		// /obj/item/reagent_containers/glass/beaker,
```

Comment:
this intentional?

Reply:
Yeah, if we add it here, we can pick up a bunch of miscellaneous things (like drug baggies) that are technically beakers, but really shouldn't be used here.